test_util/compareRestart.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+   Copyright 2016 Statoil ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#include <opm/output/eclipse/EclFilesComparator.hpp>
+
+#include <iostream>
+
+#include <ert/ecl/ecl_grid.h>
+
+void printHelp();
```

Comment:
nitpicking, but you could

1) move this above main so you don't have to prototype it
2) make it static void printHelp(). a static function in a source file is hidden from the linker, so you avoid possible name conflicts (not that anybody else should have claimed printHelp() but for educational purposes..)